From 43acf2f90b8aabe15ad5b1fac23cbb483f5ebd90 Mon Sep 17 00:00:00 2001
From: Epikhin Mikhail <schizophrenia@yandex-team.ru>
Date: Wed, 8 Apr 2020 14:19:31 +0300
Subject: [PATCH] MDB-7789: Update oozie to 5.2.0 * backport BIGTOP-3099 for
 oozie-5.2 * Exclude org.mortbay.jetty old libraries from server pom.xml

---
 pom.xml        | 18 ++++++++++++++++++
 server/pom.xml | 28 ++++++++++++++++++++++++++++
 2 files changed, 46 insertions(+)

diff --git a/pom.xml b/pom.xml
index f161b53e4..11e59088d 100644
--- a/pom.xml
+++ b/pom.xml
@@ -551,6 +551,10 @@
                         <groupId>joda-time</groupId>
                         <artifactId>joda-time</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.logging.log4j</groupId>
+                        <artifactId>log4j-1.2-api</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
 
@@ -1016,6 +1020,10 @@
                         <groupId>hadoop</groupId>
                         <artifactId>core</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.logging.log4j</groupId>
+                        <artifactId>log4j-1.2-api</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
 
@@ -1124,6 +1132,10 @@
                         <artifactId>zookeeper</artifactId>
                         <groupId>org.apache.zookeeper</groupId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.logging.log4j</groupId>
+                        <artifactId>log4j-1.2-api</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
 
@@ -1506,6 +1518,12 @@
                 <artifactId>hive-exec</artifactId>
                 <version>${hive.version}</version>
                 <classifier>${hive.classifier}</classifier>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.logging.log4j</groupId>
+                        <artifactId>log4j-1.2-api</artifactId>
+                    </exclusion>
+                </exclusions>
             </dependency>
 
             <!-- examples -->
diff --git a/server/pom.xml b/server/pom.xml
index 40fbb33c6..8f34838f7 100644
--- a/server/pom.xml
+++ b/server/pom.xml
@@ -90,6 +90,10 @@
                     <artifactId>jetty-util</artifactId>
                 </exclusion>
                 <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>jetty-sslengine</artifactId>
+                </exclusion>
+                 <exclusion>
                     <groupId>javax.servlet.jsp</groupId>
                     <artifactId>jsp-api</artifactId>
                 </exclusion>
@@ -129,6 +133,30 @@
                     <groupId>org.eclipse.jetty.aggregate</groupId>
                     <artifactId>jetty-all</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>jetty-util</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>jetty-util</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>jsp-2.1</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>jsp-api-2.1</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>servlet-api-2.5</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.mortbay.jetty</groupId>
+                    <artifactId>jetty-sslengine</artifactId>
+                </exclusion>
                 <exclusion>
                     <groupId>javax.servlet.jsp</groupId>
                     <artifactId>jsp-api</artifactId>
-- 
2.21.1 (Apple Git-122.3)

